templates/user/overview/header.tmpl

```diff
@@ -23,10 +23,15 @@
 				{{svg "octicon-repo"}} {{.locale.Tr "user.repositories"}}
 			</a>
 			{{if (not .UnitPackagesGlobalDisabled)}}
-				<a href="{{.ContextUser.HTMLURL}}/-/packages" class="{{if .IsPackagesPage}}active{{end}} item">
+				<a href="{{.ContextUser.HTMLURL}}/-/packages" class="{{if .IsPackagesPage}}active {{end}}item">
 					{{svg "octicon-package"}} {{.locale.Tr "packages.title"}}
 				</a>
 			{{end}}
+			{{if .IsRepoIndexerEnabled}}
+				<a href="{{.ContextUser.HTMLURL}}/-/code" class="{{if .IsCodePage}}active {{end}}item">
```

Comment:
This code also seems to be newly written.

Reply:
it's actually coming from few lines above (changed them also as would be stupid to leave two different approaches with 4 lines in between